src/editors/editorsController.ts

```diff
@@ -196,17 +196,17 @@ export default class EditorsController {
       });
 
       // Save document changes to active editor.
-      activeEditor?.document.save();
+      await activeEditor?.document.save();
```

Comment:
Looks like here we could wait for saving before showing a successful message modal.